config/rabbitmq.php

```diff
@@ -39,6 +39,7 @@
         'durable'     => env('RABBITMQ_QUEUE_DURABLE', true),
         'exclusive'   => env('RABBITMQ_QUEUE_EXCLUSIVE', false),
         'auto_delete' => env('RABBITMQ_QUEUE_AUTODELETE', false),
+        'arguments'   => json_decode(env('RABBITMQ_QUEUE_ARGUMENTS', '{}'), 1),
```

Comment:
It just looks strange to `json_decode` string in configuration. I suggest to move it to `RabbitMQQueue` constructor. What do you think?